webapp/javascript/redux/reducers/settings.ts

```diff
@@ -34,15 +35,29 @@ type ApiKeysState = {
   data?: APIKeys;
 };
 const apiKeysInitialState: ApiKeysState = { type: 'pristine', data: undefined };
+let reloadApiKeysAbortController: AbortController | undefined;
```

Comment:
The problem with having a single `AbortController` per action is that some actions are called twice (eg comparison view)

![image](https://user-images.githubusercontent.com/6951209/167913946-93c4c430-4824-4dc7-ba43-7ed861ad4049.png)

Only thing that I can think now is split those up (making 2 different actions), or add some custom logic per action (storing 2 abort controlelrs for example)

Reply:
let's see when we get to it